lib/depject/message/html/markdown.js

```diff
@@ -108,18 +109,3 @@ function LoadingBlobHook (hasBlob) {
     }
   }
 }
-
-function LargeEmojiHook () {
```

Comment:
does this change remove the large emoji feature?

is it possible to keep this feature by changing this code to search for `p > span.Emoji:onlyChild`, add a `-large` class to matches, and in the css add a `.Emoji .-large` class with a larger `font-size` (`24px`, double the standard font size)?

Reply:
as in, i like the feature of making standalone emoji on a single line large, without having to use an `h1`, since (as you've described before) that messes with the document semantics.

Reply:
Good idea! I didn't consider how simple that would be with font-size, thanks for the suggestion! I've just pushed a commit to re-add the large emoji.